packages/@aws-cdk/aws-synthetics/lib/schedule.ts

```diff
@@ -13,8 +13,11 @@ export class Schedule {
   }
 
   /**
-   * Construct a schedule from a literal schedule expression. The expression must be in a `rate(number units)` format.
-   * For example, `Schedule.expression('rate(10 minutes)')`
+   * Construct a schedule from a literal schedule expression. The expression must be in a `rate(number units)` or `cron(0 12 * * * *)` format.
```

Comment:
cron has a complicated syntax that we shouldn't try to reproduce it here. Please also add a link to https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_cron.html
```suggestion
   * Construct a schedule from a literal schedule expression. The expression must be in a `rate(number units)` or cron format.
```